### PR TITLE
[incubator-kie-issues#1596] Adding Business Calendar Example

### PR DESCRIPTION
**Pull request by Abhitocode**
Closes: https://github.com/apache/incubator-kie-issues/issues/1596

Add an example ensuring business calendar functionality to kogito-quarkus-examples. 
The example should include a simple process that uses a business calendar. The example should contain a good README with clear step by step instructions, and has to incorporate all supporting resources required. The priority would be having this example for Quarkus.

Note: PR still in progress, files needs to be updated.

### CODE REVIEW

**Comment by gitgabrio** [comment]
Hi @Abhitocode Thanks for the PR!

Some comments:

1. please declare the newly added `business-calendar-example` module to parent pom
2. please add a pom.xml to this new module
3. please add some integration tests - if this example is using something already tested in kogito-runtimes, you may use the same approach, even if behind a rest invocation
4. is this correct -> `business.cal.timezone= system default timezone..` ?

File: kogito-quarkus-examples/process-business-calendar-example/src/main/resources/org/application.properties
Comment (gitgabrio) [inline kogito-quarkus-examples/process-business-calendar-example/src/main/resources/org/application.properties:1]: This has to be under "/resources", together with calendar.properties

**Comment by kie-ci3** [comment]
**PR job** `#5` was: **UNSTABLE**
Possible explanation: This should be test failures


<details>
<summary><b>Reproducer</b></summary>



build-chain build full_downstream  -f 'https://raw.githubusercontent.com/${AUTHOR:apache}/incubator-kie-kogito-pipelines/${BRANCH:main}/.ci/buildchain-config-pr-cdb.yaml' -o 'bc' -p apache/incubator-kie-kogito-examples -u https://github.com/apache/incubator-kie-kogito-examples/pull/2031 --skipParallelCheckout

NOTE: To install the build-chain tool, please refer to https://github.com/kiegroup/github-action-build-chain#local-execution

</details>


Please look here: https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-examples-pr/job/PR-2031/5/display/redirect

**Test results:**
- PASSED: 391
- FAILED: 4

Those are the test failures: 
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-examples-pr/job/PR-2031/5/testReport/org.kie.kogito.examples/OrdersProcessIT/testOrderProcessWithError/">org.kie.kogito.examples.OrdersProcessIT.testOrderProcessWithError</a></summary>
Process instance with id 7a6449f6-7784-445c-b8d4-a2d0e58a1842 failed becuase of Process instance with id 7a6449f6-7784-445c-b8d4-a2d0e58a1842 failed becuase of [demo.orders:7a6449f6-7784-445c-b8d4-a2d0e58a1842 - Dump order:[uuid=_0617D7DF-047A-4EC4-85E7-E201D640F4F5]] -- Cannot invoke "String.toUpperCase()" because "<local2>" is null
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-examples-pr/job/PR-2031/5/testReport/org.kie.kogito.examples/OrdersRestIT/testOrdersWithErrorRest/">org.kie.kogito.examples.OrdersRestIT.testOrdersWithErrorRest</a></summary>
1 expectation failed.<br/>JSON path [0].id doesn't match.<br/>Expected: is "57e0a65a-6b46-4be2-a618-4722b2cf531c"<br/>  Actual: null<br/>
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-examples-pr/job/PR-2031/5/testReport/org.kie.kogito.examples.demo/OrdersProcessIT/testOrderProcessWithError/">org.kie.kogito.examples.demo.OrdersProcessIT.testOrderProcessWithError</a></summary>
Process instance with id bae56bf0-7a4d-4602-a8a5-44c6b1905afd failed becuase of Process instance with id bae56bf0-7a4d-4602-a8a5-44c6b1905afd failed becuase of [demo.orders:bae56bf0-7a4d-4602-a8a5-44c6b1905afd - Dump order:[uuid=_0617D7DF-047A-4EC4-85E7-E201D640F4F5]] -- Cannot invoke "String.toUpperCase()" because "approver" is null
</details>
<details>
<summary><a href="https://ci-builds.apache.org/job/KIE/job/kogito/job/main/job/pullrequest_jobs/job/kogito-examples-pr/job/PR-2031/5/testReport/org.kie.kogito.examples.demo/OrdersRestIT/testOrdersWithErrorRest/">org.kie.kogito.examples.demo.OrdersRestIT.testOrdersWithErrorRest</a></summary>
1 expectation failed.<br/>JSON path [0].id doesn't match.<br/>Expected: is "f698f2ae-0910-4b29-9473-4d3a9f6027a3"<br/>  Actual: null<br/>
</details>

File: kogito-quarkus-examples/process-business-calendar-example/src/test/resources/calendar.properties [hunk 1]
```diff
@@ -0,0 +1,12 @@
+business.end.hour=24
+business.hours.per.day=24
+business.start.hour=0
+business.holiday.date.format=yyyy-MM-dd
+business.days.per.week =7
+business.cal.timezone= system default timezone..
```
Comment (yesamer) [inline kogito-quarkus-examples/process-business-calendar-example/src/test/resources/calendar.properties:6]: @Abhitocode I guess this property should be commented on, right?

Comment (Abhitocode) [reply kogito-quarkus-examples/process-business-calendar-example/src/test/resources/calendar.properties:6]: @yesamer Yes, you're right, missed on it, will update in the reopened pr https://github.com/apache/incubator-kie-kogito-examples/pull/2036

**Pull request closed without merging**